src/main/java/uk/gov/hmcts/reform/blobrouter/util/ZipVerifiers.java

```diff
@@ -112,15 +98,15 @@ static void verifyFileNames(Set<String> fileNames) {
         }
     }
 
-    private static void verifySignature(String publicKeyBase64, Map<String, byte[]> entries) {
-        verifySignature(
+    private static boolean verifySignature(String publicKeyBase64, Map<String, byte[]> entries) {
+        return verifySignature(
             publicKeyBase64,
             entries.get(DOCUMENTS_ZIP),
             entries.get(SIGNATURE_SIG)
         );
     }
 
-    public static void verifySignature(String publicKeyBase64, byte[] data, byte[] signed) {
+    public static boolean verifySignature(String publicKeyBase64, byte[] data, byte[] signed) {
```

Comment:
I'd say it's not related to removing the config entry.
Also, why return boolean if only true is ever returned? (if signature is invalid, in current implementation, an exception is thrown)

Reply:
reverted the change